app/templates/index.html

```diff
@@ -0,0 +1,57 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+<div class="max-w-screen-xl mx-auto">
+    <div class="relative z-10 pb-8 bg-white sm:pb-16 md:pb-20 lg:max-w-2xl lg:w-full lg:pb-28 xl:pb-32">
+        <!-- Angled image on the right -->
+        <svg class="hidden lg:block absolute right-0 inset-y-0 h-full w-48 text-white transform translate-x-1/2"
+            fill="currentColor" viewBox="0 0 100 100" preserveAspectRatio="none">
+            <polygon points="0,0 50,0 100,100 50,100" />
+        </svg>
+
+        <main class="mt-10 mx-auto max-w-screen-xl px-4 sm:mt-12 sm:px-6 md:mt-16 lg:mt-20 lg:px-8 xl:mt-28">
+            <div class="sm:text-center lg:text-center">
+                <h2
+                    class="text-4xl tracking-tight leading-10 font-extrabold text-gray-900 sm:text-5xl sm:leading-none md:text-6xl">
+                    PyLendar
+                    <br class="xl:hidden">
+                </h2>
+                <p
+                    class="mt-3 text-base text-gray-500 sm:mt-5 sm:text-lg sm:max-w-xl sm:mx-auto md:mt-5 md:text-xl lg:mx-0">
+                    Open Source Calendar built with Python
+                </p>
+                <div class="mt-3 text-base text-gray-500 sm:mt-5 sm:text-lg sm:max-w-xl sm:mx-auto md:mt-5 md:text-xl lg:mx-0">
+                    {% if quote %}
+                        {% if not quote.author%}
+                            <p><i>"{{ quote.text }}"</i></p>
+                        {% else %}
+                            <p><i>"{{ quote.text }}"</i> <span style="font-size: small;">&nbsp; \ {{quote.author}}</span></p>
+                        {% endif %}
+                    {% endif %}
+                </div>
+                <div class="mt-5 sm:mt-8 sm:flex sm:justify-center lg:justify-center">
+                    <div class="mt-6 flex justify-center">
+                        <a href="{{ url_for('calendar') }}"
+                            class="flex rounded-full items-center py-2 px-6 bg-gradient2 focus:outline-none shadow-lg text-white text-2xl"
+                            type="button">
+                            <span>Get started</span>
+                        </a>
+                    </div>
+                </div>
+            </div>
+        </main>
+    </div>
+    <div class="lg:absolute lg:inset-y-0 lg:right-0 lg:w-1/2">
```

Comment:
There are 4 <div>s that capture only 1 <img> - maybe it can be simplify ?

Reply:
Sure, will be done